lib/pure/parsecfg.nim

```diff
@@ -107,6 +107,70 @@
 ##     var dict = loadConfig("config.ini")
 ##     dict.delSectionKey("Author","email")
 ##     dict.writeConfig("config.ini")
+## 
+## Supported INI File structure
+## ----------------------------
+## The examples below are supported:
+##
+
+# taken from https://docs.python.org/3/library/configparser.html#supported-ini-file-structure
+runnableExamples:
+  import streams
+
+  var dict = loadConfig(newStringStream("""[Simple Values]
```

Comment:
nit (can be done after PR): newline after `"""` is more readable (and equivalent; it's a feature of triple string literals)